tailwind.config.ts

```diff
@@ -1,20 +1,76 @@
 import type { Config } from "tailwindcss";
 
-const config: Config = {
+module.exports = {
   content: [
-    "./src/pages/**/*.{js,ts,jsx,tsx,mdx}",
-    "./src/components/**/*.{js,ts,jsx,tsx,mdx}",
+      "./src/**/**/*.{js,ts,jsx,tsx,mdx}",
     "./src/app/**/*.{js,ts,jsx,tsx,mdx}",
   ],
+
+  safelist: ["-left-[6px]","-left-[18px]","-left-[12px]"],
+
   theme: {
+   
+    screens: {
+      sm: '640px',
+      md: '768px',
+      lg: '1024px',
+      xl: '1280px',
+      xxl: '1440px',
+      '2xxl': '1680px',
+    },
     extend: {
+       colors: {
+      white: 'rgb(var(--white) / <alpha-value>)',
+      black: 'rgb(var(--black) / <alpha-value>)',
+      success: 'rgb(var(--success) / <alpha-value>)',
+      warning: 'rgb(var(--warning) / <alpha-value>)',
+      danger: 'rgb(var(--danger) / <alpha-value>)',
+      info: 'rgb(var(--info) / <alpha-value>)',
+      secondary: 'rgb(var(--secondary) / <alpha-value>)',
+      gray: 'rgb(var(--gray) / <alpha-value>)',
+      pink: 'rgb(var(--pink) / <alpha-value>)',
+      blue: 'rgb(var(--blue) / <alpha-value>)',
+
+      primary: {
+        50: 'var(--primary-50)',
+        60: 'var(--primary-60)',
```

Comment:
with shadcn you have used the custom theming you might have to see with the kits components